Tools/AP_Bootloader/board_types.txt

```diff
@@ -335,6 +335,9 @@ AP_HW_MFT-SEMA100                    2000
 
 AP_HW_SakuraRC-H743                  2714
 
+# IDs 4000-4040 reserved for Karshak Drones
```

Comment:
Please restrict to 10 IDs at a time.

You can always come back for more later.

Reply:
Mr Peter Barker,
Changes done. Please review it.
Thank You.